### PR TITLE
Fix deprecated setting in Convex's tests for GLPK

### PR DESCRIPTION
**Pull request by ericphanson**

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -56,7 +56,7 @@ end
 
     @testset "GLPK" begin
         run_tests(; exclude=[r"exp", r"sdp", r"socp"]) do p
-            solve!(p, () -> GLPK.Optimizer(msg_lev = GLPK.OFF))
+            solve!(p, () -> GLPK.Optimizer(msg_lev = GLPK.GLP_MSG_OFF))
```
Comment (odow) [inline test/runtests.jl:59]: You probably need something like the following.
```Julia
function optimizer()
    model = GLPK.Optimizer()
    MOI.set(model, MOI.Silent(), true)
    return model
end

solve!(p, optimizer)
```

**Review by odow — APPROVED** [review]

**Comment by ericphanson** [comment]
Ok, I used Convex's `silent_solver` keyword argument which basically does the same thing (well, it calls `MOI.set(model, MOI.Silent(), true)` on the populated model, not the optimizer, but I think it should work the same). Too bad keyword arguments for settings are deprecated, that was convenient.

**Comment by odow** [comment]
You can also use `MOI.OptimizerWithAttributes(GLPK.Optimizer, MOI.Silent() => true)`. We removed kwargs because they don't separate the options from the constructor well-enough, and we're trying to push towards a standardized way of declaring options amongst solvers.

**Comment by ericphanson** [comment]
Ok, thanks for the explanation! I probably need to update SDPAFamily at some point then.

**Pull request merged**